src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md

```diff
@@ -159,12 +160,13 @@ Valid values are:
 - free
 - standard
 - premium
+- PerGB2018
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
-Accepted values: free, standard, premium, pernode, standalone
+Accepted values: free, standard, premium, pernode, standalone, PerGB2018
```

Comment:
@MGoedtel the `Sku` parameter has a [`ValidateSet`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Workspaces/NewAzureOperationalInsightsWorkspaceCommand.cs#L45), which means it only accepts values listed in the provided array (and these values above are generated from this list). We should ensure that the version of the SDK being used for OperationalInsights supports the `PerGB2018` value before adding it to the list of accepted values.

@haitch would you mind checking this out?

Reply:
@cormacpayne I am no longer working on this project anymore.

Reply:
@xizhamsft would you mind taking a look?

Reply:
@cormacpayne - Understood and standing by.